_inc/lib/core-api/wpcom-endpoints/gutenberg-available-extensions.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * Gutenberg: List Available Gutenberg Extensions (Blocks and Plugins)
+ *
+ * [
+ *   { # Availabilty Object. See schema for more detail.
+ *     available:          (boolean) Whether the extension is available
+ *     unavailable_reason: (string)  Human readable reason for the extension not being available
```

Comment:
Human readable reason for the extension not being available 
is not really true. since reason usally looks like `missing_module` or `missing_plan`